<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diagrafi_lexis</class>
 <widget class="QDialog" name="diagrafi_lexis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>60</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_akuro">
      <property name="text">
       <string>Άκυρο</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_diag">
      <property name="text">
       <string>Διαγραφή</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEditEuresiGb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Εύρεση Αγγλικής:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEuresiGr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Εύρεση Ελληνικής:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
